docs/getting_started.rst

```diff
@@ -340,3 +406,85 @@ After this, we can close the transport used by the session.
 .. code-block:: C
 
     uxr_close_udp_transport(&transport);
+
+.. _deployment_example:
+
+Deployment example
+^^^^^^^^^^^^^^^^^^
+
+This section is devoted to illustrate how to deploy a system using *eProsima Micro XRCE-DDS* in a real environment.
+An example of this can be found in the :code:`examples/Deployment` folder.
+
+The tutorials above are based on *all in one* examples, that is, examples that create entities, publish or subscribe and then delete the resources.
+One possible real purpose of this consists in differentiating the logic of `creating entities` and the actions of `publishing and subscribing`.
+It can be done by creating two differents *Clients*, one in charge of configuring the entities in the *Agent*, which runs once,
+only for creating the entities at **compile-time**, and other/s that log(s) to the same session as the first *Client*
+(sharing the entities) and only publish(es) or subscrib(es) to data.
+
+This allows creating *Clients* in a real scenario with the only purpose of sending and receiving data.
+The concept of `profiles` allows building the *Client* library only with the chosen behavior
+(only to publish or to subscribe, for example).
+See :ref:`micro_xrce_dds_client_label` for more information.
+
+The diagram below shows an example of how to configure the environment using a `configurator client`.
+
+Initial state
+-------------
+
+    .. image:: images/deployment_0.svg
+        :width: 600 px
+        :align: center
+
+The environment contains two *Agents* (it's perfectly possible to use only one *Agent* too), and two *Clients*,
+one for publishing and another for subscribing.
+
+
+Publisher configuration
+-----------------------
+
+    .. image:: images/deployment_1.svg
+        :width: 600 px
+        :align: center
+
+In this state a `configurator client` is connected to the *Agent* `A` with the `client key` that will be used by the future `publisher client` (0xAABBCCDD).
+Once a session is logged in, the `configurator client` creates all the necessary entities for the `publisher client`.
+This implies the creation of `participant`, `topic`, `publisher`, and `datawriter` entities.
+These entities have a representation as DDS entities, and can be reached from the DDS world, that is,
+any `subscriber DDS entity` could already be listening to topics if it matches with such `publisher DDS entity` through the `DDS` world.
+
+Publisher
+---------
+    .. image:: images/deployment_2.svg
+        :width: 600 px
+        :align: center
+
+Then, the `publisher client` is connected to the *Agent* `A`.
+This *Client* logs in a session with its *Client* key (0xAABBCCDD).
+
+.. TODO: check the above sentence: "logs in a session". And in general the use of "log" and "login".
```

Comment:
@pablogs9 @jamoralp

Reply:
I have check [the standard](https://www.omg.org/spec/DDS-XRCE/1.0/PDF) and there is no reference neither to `login` nor `log in`.

So if the concept of `log in` fits for you as "create a session in the agent", just accept this suggestion.

```suggestion
```